#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()
gen.add("distance_threshold", int_t, 0, "the number used to determine whether the point is neighboring or not", 10, 0, 1000)
gen.add("point_color_threshold", int_t, 0, "the number used to detemine if the point belongs to cluster", 6, 0, 100)
gen.add("region_color_threshold", int_t, 0, "the number used when the merging process takes place", 5, 0, 100)
gen.add("min_cluster_size", int_t, 0, "the min number of the cluster size", 600, 0, 1000)


exit (gen.generate (PACKAGE, "jsk_pcl_ros", "ColorBasedRegionGrowingSegmentation"))
